iosApp/KmpGithubMVVM/KmpGithubMVVM.xcodeproj/project.pbxproj

```diff
@@ -352,6 +352,14 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				"OTHER_LDFLAGS[arch=*]" = (
+					"$(inherited)",
+					"-ObjC",
+					"-l\"c++\"",
+					"-framework",
+					"\"shared\"",
+					"-lsqlite3",
```

Comment:
see https://github.com/cashapp/sqldelight/issues/1442#issuecomment-523435492 for details